int32 count
---
recurrent_connectivity/Ridge[] ridges